plugins/linuxplugin/ifplugin/linuxcalls/ip_linuxcalls.go

```diff
@@ -38,6 +40,20 @@ func AddInterfaceIP(ifName string, addr *net.IPNet, timeLog measure.StopWatchEnt
 	if err != nil {
 		return err
 	}
+
+	exAddrList, err := netlink.AddrList(link, netlink.FAMILY_ALL)
+	if err != nil {
+		return err
+	}
+
+	// The check is basically because of link local addresses which sometimes cannot be reassigned
+	for _, exAddr := range exAddrList {
+		if bytes.Compare(exAddr.IP, addr.IP) == 0 {
+			log.Debugf("Cannot assign %v to interface %v, IP already exists", addr.IP.String(), ifName)
+			return nil
```

Comment:
Should this really `return` and not `continue`?

Reply:
it should continue, thanks